### ISSUE TITLE
"esbuild-svelte" didn't set a resolve directory

### ISSUE THREAD

**Issue by Valexr** [EMH333/esbuild-svelte#69]
When i use url() in css like:

```css
content: url('img/img.jpg');
/* or */ 
background-image: url('img/img.jpg');
```
i have a error:
```sh
> fakecss:/path/to/Component.esbuild-svelte-fake-css:1:458: error: Could not resolve "img/img.jpg" (the plugin "esbuild-svelte" didn't set a resolve directory)
```

**To Reproduce**
[REPO](https://github.com/Valexr/Compary)

**Expected behavior**
bundling url() to css 🤷🏻‍♂️

- esbuild-svelte version: latest
- esbuild version: latest
- Svelte preprocessors used (if any): sass

**Comment by EMH333** [comment]
Yep that's a bug! I'm a bit swamped this week but I'll get on this as soon as a can!

For future @EMH333 / anyone tackling this: Need to return [`resolveDir`](https://esbuild.github.io/plugins/#resolve-results) in the `onResolve` callback, however current code isn't set up to do that. Can split the file path and return that?

**Comment by Valexr** [comment]
> Yep that's a bug! I'm a bit swamped this week but I'll get on this as soon as a can!
> 
> For future @EMH333 / anyone tackling this: Need to return [`resolveDir`](https://esbuild.github.io/plugins/#resolve-results) in the `onResolve` callback, however current code isn't set up to do that. Can split the file path and return that?

if in resolveDir true absolute path it's working

**Comment by EMH333** [comment]
Fixed in `v0.5.5`

**Comment by Valexr** [comment]
What is this ```require(";`);``` in https://unpkg.com/browse/esbuild-svelte@0.5.5/dist/index.js `line:23`

**Comment by EMH333** [comment]
Oh wow! That appears to be a bug in `rewrite-imports`. I'll follow up later to confirm. Fix released in `v0.5.6`

**Comment by Valexr** [comment]
tx for quick feedback 👍🏻 testing
in `resolveDir: /path/to/component/where/make/import` ?
